lib/menu.rb

```diff
@@ -0,0 +1,14 @@
+module Menu
+
+  def menu
+    dish_list = {
+      'pepperoni pizza' => 5.00,
+      'spaghetti pomodoro' => 6.00,
+      'spaghetti carbonara' => 7.50
+    }
+
+    dish_list.each do |dish, price|
+      puts "#{dish} => #{'%.2f' % price}"
```

Comment:
Style/FormatString: Favor `format` over `String#%`.<br>Rails/Output: Do not write to stdout. Use Rails' logger if you want to log.